docs/zh_cn/tutorials/qcloud.md

```diff
@@ -285,22 +285,14 @@ sudo juicefs umount /mnt/jfs
 
 ## 开机自动挂载
 
-如果你不想每次重启系统都要重新手动挂载 JuiceFS 存储，可以设置自动挂载文件系统。
-
-首先，需要将  `juicefs` 客户端重命名为 `mount.juicefs` 并复制到 `/sbin/` 目录：
-
-```shell
-sudo cp juice/juicefs /sbin/mount.juicefs
+可以用 `juiefs mount --update-fstab` 直接设置出自动挂载，例如：
```

Comment:
This will be available in 1.1 release, but user still read the latest docs, so we should add docs for new feature without replacing the old ways.

For these tutorials, we don't need to change them (keep them simple).

Reply:
the old ways are preserved in guide/mount_at_boot.md, I can add minimum version requirements here (and remove after 1.1 has been widely adopted), would this suffice? 

I think putting "the old way" here might just be too much content, after all, there's a dedicated chapter on this topic.

Reply:
> For these tutorials, we don't need to change them (keep them simple).

`--update-fstab` is the simplest way to approach auto-mount, shouldn't we recommend its usage in all tutorials? if version inconsistency is the main issue here, can I postpone this docs update until after 1.1 release?